packages/http-client/src/interfaces.ts

```diff
@@ -1,83 +1,84 @@
-import http = require('http')
+import * as http from 'http'
+import * as https from 'https'
 
-export interface IHeaders {
-  [key: string]: any
+export interface Headers {
+  [header: string]: string
```

Comment:
I looked at existing usage in `http-client` and other toolkit packages and we always use string values for headers.  Node's `http` module tends to use `string | string[] | undefined` for header values, but that would have required more code changes to handle where headers are consumed in `http-client`.